descartes_planner/src/planning_graph.cpp

```diff
@@ -1004,19 +1058,17 @@ double PlanningGraph::linearWeight(JointTrajectoryPt start, JointTrajectoryPt en
     std::vector<double> end_vector = joint_poses_end[0];
     if (start_vector.size() == end_vector.size())
     {
+      if (end.getTiming().isSpecified() && !robot_model_->isValidMove(start_vector, end_vector, end.getTiming().upper_))
+      {
+        return MAX_EXCEEDED_PENALTY;
```

Comment:
Am I reading this correctly...if the timing is specified and the move is not valid, we set the max penalty?  Shouldn't we prune the edge?  Or is this a "magic" value that indicates we should prune the edge?